src/algebra/algebra/basic.lean

```diff
@@ -463,6 +463,8 @@ variables (R : Type u) (M : Type v) [comm_semiring R] [add_comm_monoid M] [modul
 instance : algebra R (module.End R M) :=
 algebra.of_module smul_mul_assoc (λ r f g, (smul_comm r f g).symm)
 
+instance : is_scalar_tower R (module.End R M) M := ⟨λ t f m, rfl⟩
```

Comment:
I think this belongs earlier, near where `module.End.apply_mul_action` is defined.

Reply:
Just beside these:

https://github.com/leanprover-community/mathlib/blob/a90f9692d06effaac57537aaea8850ca09872848/src/algebra/module/linear_map.lean#L791-L795

Reply:
I found it moments before you wrote this 🎉